inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/api/ClientContext.java

```diff
@@ -17,19 +17,27 @@
 
 package org.apache.inlong.sdk.sort.api;
 
+import org.apache.inlong.common.metric.MetricRegister;
 import org.apache.inlong.sdk.sort.entity.InLongTopic;
-import org.apache.inlong.sdk.sort.stat.SortClientStateCounter;
-import org.apache.inlong.sdk.sort.stat.StatManager;
+import org.apache.inlong.sdk.sort.metrics.SortSdkMetricItem;
+import org.apache.inlong.sdk.sort.metrics.SortSdkMetricItemSet;
+
+import java.util.HashMap;
+import java.util.Map;
 
 public abstract class ClientContext implements Cleanable {
 
     protected final SortClientConfig config;
 
-    protected final StatManager statManager;
+    protected final String sortTaskId;
+
+    protected final SortSdkMetricItemSet metricItemSet;
 
-    public ClientContext(SortClientConfig config, MetricReporter reporter) {
+    public ClientContext(SortClientConfig config) {
         this.config = config;
-        this.statManager = new StatManager(this, reporter);
+        this.sortTaskId = config.getSortTaskId();
```

Comment:
This PR has too many changes. Is it necessary to add some UTs to verify them?

Reply:
This PR includes modification of #6944. 
I will rebase it when #6944 is merged.